docker/ray/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM rayproject/base-deps
+ARG WHEEL_PATH
+ARG WHEEL_NAME
+COPY $WHEEL_PATH $WHEEL_NAME
```

Comment:
Why make this a variable? it's the Ray dockerfile. Just `COPY` the wheel (perhaps from the local directory, which may require a `cp` above, or just reference the relative path) and install it.

The fact that, below, it's $WHEEL_NAME[all] shows that the abstraction of $WHEEL_NAME is leaky (ie, there definitely exists an [all] extra package -- which isn't true for all wheels)

Just install ray in the ray docker file. Then other auto-scalable things with custom environments start with `FROM rayproject/ray`